pom.xml

```
@@ -6,9 +6,8 @@
 	<parent>
 		<groupId>life.qbic</groupId>
 		<artifactId>portlet-parent-pom</artifactId>
-		<version>1.3.0</version>
+		<version>2.1.0</version>
```

the most recent one is 2.2.5, any reason for not using it?

---

No we could update I guess.

---

```suggestion
		<version>2.2.5</version>
```
Feel free to accept so we both get credit @sven1103